Tools/LambdaTestTool-v2/tests/Amazon.Lambda.TestTool.IntegrationTests/ApiGatewayResponseExtensionsTestsManual.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using Amazon.Lambda.APIGatewayEvents;
+using Microsoft.AspNetCore.Http;
+using System.Text.Json;
+using Amazon.Lambda.TestTool.Extensions;
+
+namespace Amazon.Lambda.TestTool.IntegrationTests
+{
+    [Collection("ApiGateway Integration Tests")]
+    public class ApiGatewayResponseExtensionsTestsManual
```

Comment:
What does the `Manual` suffix mean here? This test will run automatically as part of integration test runs.

Reply:
ill rename this to something better. its not part of the ApiGatewayResponeTestCases list and the test cases are separate. maybe `AdditionalTests`